app/src/main/java/edu/grinnell/grinnell_publications_android/Activities/MainActivity.java

```diff
@@ -5,12 +5,14 @@
 import android.support.design.widget.NavigationView;
 import android.support.v4.app.Fragment;
 import android.support.v4.app.FragmentTransaction;
+import android.support.v4.view.GravityCompat;
```

Comment:
Change this to `import static android.support.v4.view.GravityCompat.START;` and move this to the bottom of the import statements